hack/make/.build-rpm/docker-engine-selinux.spec

```diff
@@ -13,20 +13,7 @@ URL: https://dockerproject.org
 Vendor: Docker
 Packager: Docker <support@docker.com>
 
-# Version of SELinux we were using
-%if 0%{?fedora} == 20
-%global selinux_policyver 3.12.1-197
-%endif # fedora 20
-%if 0%{?fedora} == 21
-%global selinux_policyver 3.13.1-105
-%endif # fedora 21
-%if 0%{?fedora} >= 22
-%global selinux_policyver 3.13.1-128
-%endif # fedora 22
-%if 0%{?centos} >= 7 || 0%{?rhel} >= 7 || 0%{?oraclelinux} >= 7
```

Comment:
We weren't special casing it to being with.
But in anycase, the policy version specified is what must be supported.
If Oracle has a separate version for 6 that we need to support, we can add it.

AFAICT, these special cases were nonsense since they all stem from the same package.
The minimum supported package version is the same for each distro version unless they have a different package.